### PR TITLE
[BUILD] Add more 16k shared lib package option on Android.mk

### PR DESCRIPTION
**Pull request by DonghakPark**
After #2699 : [[ BUILD ] Add 16K shared lib package option for Android](https://github.com/nnstreamer/nntrainer/pull/2699/commits/309c8449358eeb4904149d043548e0913821271e)

```
Android encourage to use 16KB package for the shared library.
This PR add the 16KB package option and also recommand to use ndk which is higher or equal version of r27.
```

add option's for all android.mk file

**Self evaluation:**
1. Build test:	 [X]Passed [ ]Failed [ ]Skipped
2. Run test:	 [X]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

File: Applications/utils/jni/Android.mk [hunk 1]
```diff
@@ -18,6 +18,7 @@ LOCAL_CXXFLAGS      += -std=c++17 -frtti -fexceptions
 LOCAL_MODULE_TAGS   := optional
 
 LOCAL_LDLIBS        := -llog -landroid -fopenmp
+LOCAL_LDFLAGS 		+= "-Wl,-z,max-page-size=16384"
```
Comment (djeong20) [inline Applications/utils/jni/Android.mk:21]: is this added intentionally?

**Review by djeong20 — APPROVED** [review]
overall, LGTM

File: Applications/utils/jni/Android.mk (hunk 1 above)
Comment (DonghakPark) [reply Applications/utils/jni/Android.mk:21]: no, thank you for finding
i will fix

**Review by skykongkong8 — APPROVED** [review]
LGTM

**Review by SeoHyungjun — APPROVED** [review]
LGTM

**Pull request merged**